week3/homework/src/actions/clearTodos.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const saveTodos = './index';
```

Comment:
I think you might be missing something :)

Reply:
Thanks! u are right! It's done!